src/MouseAdapter.js

```diff
@@ -309,20 +383,196 @@ define(function(require) {
         _onMouseWheel: function(event) {
             // If we haven't started a mouse wheel for a bit, publish a start event.
             if (!this._wheeling) {
-                this.onMouseWheelStart.dispatch([{
-                    distance: { x: 0, y: 0 },
-                    source: event
-                }]);
-                this._wheeling = true;
+                this._dispatchMouseWheelStart(event);
             }
 
             // Normalize the mouse wheel event and publish.
             var normalizedEvent = normalizeEvent(event);
+            this._detectUserScroll(normalizedEvent);
             this.onMouseWheel.dispatch([normalizedEvent]);
 
             // Debounce mouse wheels so that an end event is published 50ms
```

Comment:
That's a good idea, i can break that into a constant.  

It's supposed to represent the delay after recieving a mouse event after which we can reasonably conclude that there are no more inertial events incoming and can then emit a mouse_end event.  It used to be 50ms, but i was seeing cases where multiple mouse_end/mouse_start events were being emitted by a single scroll gesture because the gap between inertial events was higher than the debouncer so i bumped it up until i no longer could reproduce this.